findbugs.properties

```diff
@@ -1,13 +1,19 @@
 category=External Analysers
 description=Provide Findbugs rules for analysis of Java projects
-archivedVersions=2.4,3.0,3.1,3.2,3.3,3.4.3,3.4.4,3.5,3.6,3.7,3.8.0,3.9.0,3.9.1,3.9.2,3.9.3,3.10.0,3.11.0,3.9.4,3.11.1
-publicVersions=4.0.0
+archivedVersions=2.4,3.0,3.1,3.2,3.3,3.4.3,3.4.4,3.5,3.6,3.7,3.8.0,3.9.0,3.9.1,3.9.2,3.9.3,3.10.0,3.11.0,3.9.4,3.11.1,4.0.0
+publicVersions=4.0.1
 
 defaults.mavenGroupId=org.sonarsource.sonar-findbugs-plugin
 defaults.mavenArtifactId=sonar-findbugs-plugin
 
+4.0.1.description=Use SpotBugs 4.1.2, sb-contrib 7.4.7, and findsecbugs 1.10.1
+4.0.1.sqVersions=[7.9,LATEST]
+4.0.1.date=2020-09-18
+4.0.1.changelogUrl=https://github.com/spotbugs/sonar-findbugs/releases/tag/4.0.1
```

Comment:
The changes in your release tag (version in pom & readme) don't match your description here.

Reply:
Sorry but I cannot get your point. Which version in the pom file?

https://github.com/spotbugs/sonar-findbugs/blob/78fe4818b3fccbd6a9fe6ca3982322abd0ec8b5b/pom.xml

Reply:
Okay, I misread the interface. Pom updated _since_ the tag. Which means that your "change log" gives 0 information about the changes. No narrative description, no list of (or link to) tickets, no commits. Can you please beef up the page this link points to with ... _something_?

Reply:
OK I've updated that, please confirm https://github.com/spotbugs/sonar-findbugs/releases/tag/4.0.1

Reply:
Better! Thanks!